guide/17-geocoding-guides/part1_what_is_geocoding.ipynb

```diff
@@ -0,0 +1,1334 @@
+{
```

Comment:
<p>For countries covered by a geocoding service, e.g. the ArcGIS World Geocoding Service, the address coverage, supported language, and relative geocoding quality will vary. The <code>geocode_coverage</code> WebMap object displayed below illustrates the various levels. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/830/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>